pages/partners_new.tsx

```diff
@@ -1,15 +1,252 @@
+                <Button variant="contained" color="secondary">
+                  Partner with us
+                </Button>
+              </Link>
+            </div>
+          </Box>
+          <Box sx={{ width: '50%' }} maxWidth={'700px'}>
+            <img
+              src={PartnerImage.src}
+              alt="Partner page graphic"
+              style={{ objectFit: 'cover' }}
+            ></img>
+          </Box>
+        </Box>
+      </Container>
+      <Container
+        style={{
+          backgroundColor: theme.palette.background.default,
+          color: designColor.black,
+          textAlign: 'center',
+          padding: '2rem',
+        }}
+        maxWidth={false}
+      >
+        <Box
+          sx={{
+            gap: '2rem',
+            display: 'grid',
+            margin: '2rem',
+          }}
+        >
+          <Typography variant="headlineLarge">Our criteria</Typography>
+
+          <Grid container spacing={2}>
+            {criteriaContent.map((item, index) => (
+              <Grid item md={3} key={index}>
+                <CardOne
+                  title={item.title}
+                  description={item.description}
+                  icon={item.icon}
+                />
+              </Grid>
+            ))}
+          </Grid>
+        </Box>
+        <Box
+          sx={{
+            gap: '2rem',
+            display: 'grid',
+            margin: '2rem',
+            marginTop: '5rem',
+          }}
+        >
+          <Typography variant="headlineLarge">How we can help</Typography>
+
+          <Grid container spacing={2}>
+            {provideContent.map((item, index) => (
+              <Grid item md={4} key={index}>
+                <CardOne
+                  title={item.title}
+                  description={item.description}
+                  icon={item.icon}
+                />
+              </Grid>
+            ))}
+          </Grid>
+        </Box>
+      </Container>
+      <Container
+        sx={{
+          backgroundColor: designColor.white,
+          color: designColor.black,
+          padding: '2rem',
+        }}
+        maxWidth={false}
+      >
+        <Box sx={{ gap: '2rem', display: 'grid', margin: '2rem' }}>
+          <Typography variant="headlineLarge">The process</Typography>
+          <ol>
+            {processContent.map((item, index) => (
+              <li key={index + 1} style={{ marginBottom: '2rem' }}>
+                <Typography
+                  variant="titleLarge"
+                  color={palette.primary.main}
+                >{`${index + 1}.`}</Typography>
+                <Typography variant="bodyLarge" mx={2}>
+                  {item}
+                </Typography>
+              </li>
+            ))}
+          </ol>
+          <div style={{ margin: '2rem' }}>
+            <Link href="/partner-application" passHref>
+              <Button variant="contained" color="primary">
+                Partner with us
+              </Button>
```

Comment:
nit: Buttons in MUI take an href prop, so you don't have to wrap them in a Link!

Reply:
Thanks!